lizaku/assignment4/assignment4.py

```diff
@@ -0,0 +1,86 @@
+import itertools
```

Comment:
Are you using itertools anywhere? I cannot find it in the file.

Reply:
No! It was in some intermediary version of the algorithm.